src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -211,7 +212,9 @@ func CreateAdmin(
 		if err != nil {
 			return nil, err
 		}
-		return sarama.NewClusterAdmin(brokers(), cfg)
+		bs := brokers()
```

Comment:
That's a harsh name for those poor brokers :laughing:

Reply:
That's what you get! haha.